airrohr-firmware/airrohr-firmware.ino

```diff
@@ -2584,15 +2595,37 @@ void display_values(const String& value_DHT_T, const String& value_DHT_H, const
 		display.setFont(Monospaced_plain_9);
 		display.setTextAlignment(TEXT_ALIGN_LEFT);
 		value_count = 0;
-		display.drawString(0, 10 * (value_count++), "Temp:" + t_value + "  Hum.:" + h_value);
+    
+		display.drawString(0, 10 * (value_count++), "Temperatuur: " + t_value + "°C");
+    display.drawString(0, 10 * (value_count++), "luchtvochtigheid: " + h_value+ " %");
+    
 		if (ppd_read) {
 			display.drawString(0, 10 * (value_count++), "PPD P1: " + value_PPD_P1);
 			display.drawString(0, 10 * (value_count++), "PPD P2: " + value_PPD_P2);
 		}
+   
 		if (sds_read) {
-			display.drawString(0, 10 * (value_count++), "SDS P1: " + value_SDS_P1);
-			display.drawString(0, 10 * (value_count++), "SDS P2: " + value_SDS_P2);
-		}
+      display.drawString(0, 10 * (value_count++), "SDS PM 10: " + value_SDS_P1 + " µg/m³");
```

Comment:
Could you please use the tab and space settings of the original source?

Reply:
I modified it but in my code the position is good.
https://snag.gy/bnmHGZ.jpg